packages/react/package.json

```diff
@@ -42,10 +42,6 @@
     "@astrouxds/astro-web-components": "0.0.18",
     "ts-jest": "^27.0.5"
   },
-  "peerDependencies": {
```

Comment:
Should I leave these in as the updated react version, or is it okay to drop them?